tests/dragonfly/sentinel_test.py

```diff
@@ -5,7 +5,11 @@
 import pytest
 import time
 import asyncio
+from datetime import datetime
+from sys import stderr
+import logging
 
+log = logging.info
```

Comment:
I'd remove this aliasing - I think explicit log level on every log is a good thing.